src/viam/sdk/CMakeLists.txt

```diff
@@ -249,3 +249,10 @@ install(FILES
 
 
 add_subdirectory(tests)
+
+set(CPACK_GENERATOR "DEB")
+set(CPACK_ARCHIVE_COMPONENT_INSTALL ON)
+set(CPACK_COMPONENTS_ALL viam-cpp-sdk_dev)
```

Comment:
Maybe this should be using `CPACK_COMPONENTS_GROUPING` to get separate debs for different roles?

Reply:
> separate debs for different roles

roles = development vs runtime?

Reply:
Yes, typically on a deb based platform, a library ships at least three packages:
- A `libfoo` package which contains the runtime role components (typically the explicitly versioned library file `libfoo.so.x.y.z` and its soname symlink `libfoo.so.x`).
- A `libfoo-dev` development role package which contains the headers and the dev time lib symlink `libfoo.so` so that `-lfoo` finds the library.
- A `libfoo-dbg` debug role package which contains debug info.